app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_session_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
+  <%= link_to "Forgot your password?", new_password_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.confirmable? && controller_name != 'confirmations' %>
+  <%= link_to "Didn't receive confirmation instructions?", new_confirmation_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks' %>
```

Comment:
While reviewing your work I noticed Your UI on the registration page and login page content  are aligned too much to the left and do not match the given wireframe design it would be nice to style your work like the given design  please align your page content to the centre and make a few adjustments so your work matches the wireframe because it says in the requirements that ,The views in devise for the registration and login have been modified by the student to match the wireframes and their styling.


This is your design: 

Your Login:

![Login ](https://github.com/ClintonYounge/My-Blog-App/assets/79506105/be5da726-6169-477c-8a01-e08612bc199c)

Your sign-up page:

![Sign up](https://github.com/ClintonYounge/My-Blog-App/assets/79506105/2d3b9544-61df-4e67-90c5-19370d126c2b)

Kind follow this design:
![Wireframe ](https://github.com/ClintonYounge/My-Blog-App/assets/79506105/db5c1709-9cfa-4b67-bd9c-7e4fd64dad04)

![Wireframe ](https://github.com/ClintonYounge/My-Blog-App/assets/79506105/db5c1709-9cfa-4b67-bd9c-7e4fd64dad04)

Please its very important Tommy attention to design and project requirements. Always kindly pay attention to detail.